packages/arb-ts/src/lib/inbox/inbox.ts

```diff
@@ -183,6 +184,38 @@ export class InboxTools {
       }
     )
 
+    if (events.length !== 0) return events
+    else if (maxedSearchRangeBlocks === maxSearchRangeBlocks) return []
+    else {
+      return await this.getEventsAndIncreaseRange(
+        bridge,
+        searchRangeBlocks * 2,
+        maxSearchRangeBlocks
+      )
+    }
+  }
+
+  /**
+   * Find the event of the latest message that can be force include
+   * @param searchRangeBlocks Defaults to 3 * 6545 ( = ~3 days) prior to the first eligble block
+   * @returns Null if non can be found.
+   */
+  public async getForceIncludeableEvent(
+    searchRangeBlocks: number = 3 * 6545
+  ): Promise<ForceInclusionParams | null> {
+    const bridge = Bridge__factory.connect(
+      this.l2Network.ethBridge.bridge,
+      this.l1Provider
+    )
+
+    // events dont become eligible until they pass a delay
+    // find a block range which will emit eligible events
+    const events = await this.getEventsAndIncreaseRange(
+      bridge,
+      100,
```

Comment:
why do we hardcode 100 here?
could be a function param with default value like in `searchRangeBlocks`
If we do the `growthRate` change for `getEventsAndIncreaseRange` we should also surface it here with a default value since this function is the user facing one

Reply:
updated